windows/ReactNativeWebView/ReactNativeWebView.vcxproj

```diff
@@ -158,19 +155,24 @@
       <Private>false</Private>
     </ProjectReference>
   </ItemGroup>
+  <Import Project="$(SolutionDir)\ReactNativeXaml.Imports.props" Condition="Exists('$(SolutionDir)\ReactNativeXaml.Imports.props')" />
```

Comment:
why is this here?

Reply:
I'll remove. Must have been added on accident.